changelog.md

```diff
@@ -1,3 +1,37 @@
+## Changes from 1.5.0 to 1.5.1
+Bugfix release
+
+### Fixed issues
+- [MARATHON-7576](https://jira.mesosphere.com/browse/MARATHON-7576) Changed default of UnreachableEnabled to (0,0)
+- TaskLauncherActor doesn't wait for in-flight tasks on stop
+- [MARATHON-7765](https://jira.mesosphere.com/browse/MARATHON-7765) Update to jar Manifest file
+- Replace custom IO with Apache Commons IO
+- Added SchedulerPlugin to enable the ability to
+- Update to the docker build process
+- [MARATHON-2520](https://jira.mesosphere.com/browse/MARATHON-2520) Improved logging around migration
+- [D1044](https://phabricator.mesosphere.com/D1044) EventStream implementation moved to Akka eventStream
+- [D1042](https://phabricator.mesosphere.com/D1042) Fix a problem with allocating port for IT by forbidding port reuse in the `PortAllocator`
+- [D1058](https://phabricator.mesosphere.com/D1058) Fixed ephemeral port range to avoid collision with executor
+- [MARATHON-7696](https://jira.mesosphere.com/browse/MARATHON-7696) variable overdueDelayCleanupInterval replaces hardcoded value
+- [D1027](https://phabricator.mesosphere.com/D1027) Fixed state which Marathon could not boot.
+- [MARATHON-7545](https://jira.mesosphere.com/browse/MARATHON-7545) Initialize RunSpecTaskProcessor and RunSpecValidator at startup to early detect misconfiguration.
+- [D974](https://phabricator.mesosphere.com/D974) Allows plugin to throw an InitializationException if it is unable to init properly.
```

Comment:
Let's make this more user facing: "Plugin configuration or initialization issues are made more obvious, potentially causing Marathon to not launch."